src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+import { Post } from '../../types/Post';
+
+type Props = {
+  postsArr: Post[],
+};
+
+export const PostList: React.FC<Props> = ({ postsArr }) => (
+  <div className="PostList">
+    {postsArr.map(onePost => (
+      <div key={onePost.id}>
+        <PostInfo {...onePost} />
```

Comment:
It's better (safer) to pass all props explicitly or pass the entire `post` object and do destructure in `PostInfo`
Try to avoid such way to pass props